### DISCUSSION TITLE
Suggested desktop environment?

### DISCUSSION THREAD

**Discussion by IcyAlmond** [linux-surface/linux-surface#661] [Q&A] [answered]
Probably applicable to the other surface laptops as well but is it suggested that i use gnome instead of kde as my desktop environment?

Surface Pro 5/2017

**Comment by aflyingpumpkin** [comment]
I have been using DWM with my surface 3 and have been loving it, though this is probably not something most people would love. This dodges the question a bit, but I would reccomend just trying out some different DEs to see what you like best. I have heard good things about Gnome and Deepin on the surface.

**Comment by SamLukeYes** [comment, marked as answer]
GNOME with Wayland provides the best touchscreen experience among all the DEs I've ever used. However, touchscreen is only a part of the whole experience. It's up to you to choose a DE or WM that you like.
